phc/easy/frame.py

```diff
@@ -1,8 +1,16 @@
-import pandas as pd
 from typing import Callable, List, Tuple
+import pandas as pd
 from phc.easy.codeable import Codeable
 
-CODE_COLUMNS = ["meta", "identifier", "extension", "telecom"]
+CODE_COLUMNS = [
+    "meta",
+    "identifier",
+    "extension",
+    "telecom",
+    "valueCodeableConcept",
```

Comment:
Added some more these columns that'll be needed for observations. It'll check to make sure the column exists before trying to expand it.